vpr/src/device/rr_graph_view.h

```diff
@@ -285,12 +295,19 @@ class RRGraphView {
     inline short edge_switch(RRNodeId id, t_edge_size iedge) const {
         return node_storage_.edge_switch(id, iedge);
     }
+    inline short edge_switch(const RREdgeId& edge) const {
+        return edge_switch_[edge];
+    }
+
     /** @brief Get the destination node for the iedge'th edge from specified RRNodeId.
      *  This method should generally not be used, and instead first_edge and
      *  last_edge should be used.*/
     inline RRNodeId edge_sink_node(RRNodeId id, t_edge_size iedge) const {
         return node_storage_.edge_sink_node(id, iedge);
     }
+    inline RRNodeId edge_sink_node(RREdgeId edge) const {
+        return edge_dest_node_[edge];
```

Comment:
@tangxifan 
We are using it here `const vtr::vector<RREdgeId, RRNodeId> edge_dest_node_;`.
to replace here.
https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/01945e933911ef932958b1b11e342d9934da1823/vpr/src/route/route_common.cpp#L1641